frontend/src/features/admin-form/share/ShareFormModal.tsx

```diff
@@ -370,11 +387,12 @@ export const ShareFormModal = ({
                         )}
                       </Stack>
                       {goLinkHelperText && (
+                        // padding on icon box to emulate padding from <Text>
                         <FormHelperText color={goLinkHelperText.color}>
-                          <Stack direction="row" align="center">
-                            <Box>{goLinkHelperText.icon}</Box>
+                          <HStack alignItems="flex-start">
```

Comment:
Ooh I'm curious, is there a difference between this and Stack? Why the change?

Reply:
Oh its just a horizontal stack 😅 HAHAHA I was just playing around with different types of flex and stacks. HStack is literally <Stack direction='row>. I can change it back hahaha

Reply:
Oh no, no need, was just wondering if there was a best practice or optimisation thing or something! Can leave it haha no worries! This comment is just for my learning 😂